### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by shrimp26** [YinHangCode/homebridge-mi-gateway-security#6]
Hello, 

Your plugin it's just perfect. But maybe you could add a return of state of the gateway to know if the alarm is on (intrusion detected) to trigger other things ?

Thank you so much for your work !

**Comment by theo-69** [comment]
That would be 👍🏼

**Comment by x23piracy** [comment]
Yeah, would be really nice.

**Comment by iJ-88** [comment]
Yeah, this would be cool. Would be great to be able to receive a notification if the alarm was triggered (via HomeKit) when not at home etc...
